modules/cnv-creating-vm.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-create-vms.adoc
+
+[id="cnv-creating-vm_{context}"]
+= Creating a virtual machine from the CLI
+
+The `spec` object of the VirtualMachine configuration file references
+the virtual machine settings, such as the number of cores and the amount
+of memory, the disk type, and the volumes to use.
+
+Attach the virtual machine disk to the virtual machine by referencing
+the relevant PVC `claimName` as a volume.
+
+[NOTE]
+====
+https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/[ReplicaSet]
+is not currently supported in {ProductName}.
+====
+
+See the Reference section for information about volume types and
```

Comment:
What Reference section?

Reply:
Changed to:

"See the virtual machine storage volume types section for information about volume types."

I will query QE on the availability and location of sample config files and if we want to include them.

Reply:
On second thought, it seems a better option to use an include to bring this table into this CLI module at the needed point. Is this allowed?

When I try using include::modules/cnv-vm-storage-volume-types.adoc[leveloffset=+1],
this is rendering with the '[leveloffset]' attribute appearing before the table.

@kalexand-rh what do you recommend? Thanks.

Reply:
See my comment below. Realized you can't call from a module. Doh!